package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "paybutton",
+  "version": "2.1.2",
+  "description": "The easiest way to accept eCash online",
+  "author": "poldridge",
+  "main": "index.js",
+  "repository": "https://github.com/PayButton/paybutton.git",
+  "license": "MIT",
+  "scripts": {
```

Comment:
Also,  would be good to have scripts that don't run `yarn` besides the ones that run it.

When developing, most of the time we won't have updated/added/removed any modules, so we don't want to be running `yarn`, which takes a considerable time to run, together with `yarn build`.